conf.py

```diff
@@ -31,8 +31,12 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
+import sys
```

Comment:
Not sure why you need the sys package... I think you can do this all without importing anything. You pass the matplotlib in as a string, so no import is required for this!